backend/onedrive/onedrive.go

```diff
@@ -137,6 +140,17 @@ Note that the chunks will be buffered into memory.`,
 			Help:     "The type of the drive (" + driveTypePersonal + " | " + driveTypeBusiness + " | " + driveTypeSharepoint + ").",
 			Default:  "",
 			Advanced: true,
+		}, {
+			Name: "disable_site_permission",
+			Help: `Disable the request for Sites.Read.All permission.
+
+If enabled, you will no longer be able to search for a SharePoint site
+when configuring drive ID.
+Enable if your organization didn't assign Sites.Read.All permission to the
+application, and your organization disallows users to consent app permission
+request on their own.`,
```

Comment:
The name of the option has the word "disable" and the documentation uses "enable." This can create confusion. I'd suggest changing the wording in the docs to something like this: "If set to true, ..." and "Set to true if ..."